### DISCUSSION TITLE
Programmatically select grid row

### DISCUSSION THREAD

**Discussion by curtic00** [vikramlearning/blazorbootstrap#930] [Q&A]
Is it possible to programmatically select a row in a grid.
I have a use case where a job record has a link to a customer record.
In the create job form, user selects a customer from a grid by clicking the row selector. Fine.

But when it comes to edit the job, I want the user to use the grid again, but will need to have the current customer pre-selected, so that the user can see who the selected customer was at job creation ....,  hope this makes sense

**Comment by benholtsclaw** [comment]
I have the same need. Essentially, we're looking for a bindable SelectedItems property. I need this property to persist through paging, sorting and filtering. It seems like there would be many uses cases for this and I'm not sure what others do to work around this limitation.

For example, if I'm creating a new classroom entry, I want to be able to display a potentially lengthy list of students in the grid. The user should be able to select students who will be assigned to the classroom. Additionally, when the user comes back to edit the classroom, the current list of students assigned to the classroom should be selected in the grid.

**Reply by MarkKamberger** [reply]
I have a PR in to add this functionality

**Comment by MarkKamberger** [comment]
Same here, needed when in editing context

**Comment by curtic00** [comment]
Cool 👍👍

> Charlie Curtis 
> Worthing, West Sussex
> BN133AB.
> MCSD, MCSA, MCRDBA, Azure administrator
> Mobile: 0777 254 5516
> LinkedIn: https://uk.linkedin.com/in/charlieecharlie
